### ISSUE TITLE
from source installation of bids-validator fails due to missing versioneer.py

### ISSUE THREAD

**Issue by boegel** [bids-standard/legacy-validator#757]
It seems like the problem reported in #707 and (supposedly) fixed in #708 has resurfaced.

Trying to install `bids-validator` 1.2.2 using the source tarballs from PyPI (https://pypi.org/project/bids-validator/1.2.2/#files) fails with:

```
$ tar xfvz bids-validator-1.2.2.tar.gz
$ cd bids-validator-1.2.2
$ pip install --prefix /tmp/example .
Processing /tmp/bids-validator-1.2.2
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/eb-0cjpGh/pip-req-build-aja1ythx/setup.py", line 3, in <module>
        import versioneer
    ModuleNotFoundError: No module named 'versioneer'
```

Indeed the `versioneer.py` file is not there in the `bids-validator-1.2.2.tar.gz` source tarball available from PyPI...

**Comment by boegel** [comment]
@duncanmmacleod Any idea how this is possible after your fix in #708?

**Comment by effigies** [comment]
Ah, figured it out. MANIFEST.in didn't get moved with #710. I'm going to make a test that ensures this doesn't happen again and submit a fix.
